src/test/java/uk/gov/hmcts/reform/sscs/job/CaseLoaderTaskletTest.java

```diff
@@ -18,34 +18,34 @@ public class CaseLoaderTaskletTest {
     private SscsCaseLoaderJob caseLoaderJob;
 
     @Mock
-    private DataMigrationJob dataMigrationJob;
+    private ProcessingVenueMigrationJob venueMigrationJob;
```

Comment:
the venueMigrationJob is in addition to the interpreter one rather than replacing it